buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchNode.java

```diff
@@ -1135,6 +1141,7 @@ private void createConfiguration() {
             baseConfig.put("node.name", nodeName);
         }
         baseConfig.put("path.repo", confPathRepo.toAbsolutePath().toString());
+        baseConfig.put("http.protocol", httpProtocol);
```

Comment:
Nit; Move this down by 1154

Reply:
Sure

Reply:
`baseConfig` would be values that would be entered into `opensearch.yml` right?

With the security plugin, the setting that controls https is this one `plugins.security.ssl.http.enabled: true`. With testclusters is it possible to install a plugin and run a setup script? 

https://github.com/opensearch-project/security/blob/main/tools/install_demo_configuration.sh#L370